apps/pm-user/src/components/molecules/quickInfoBar/QuickInfoBar.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Segmented } from '../../atoms';
+import { SEGMENT_OPTIONS } from './constants';
+import { useHistory } from 'react-router-dom';
+import styles from './quickInfoBar.module.scss';
+
+const QuickInfoBar = () => {
+  const history = useHistory();
+
+  //redirecting on changing the value
+  function handleRedirectOnClick(value) {
+    // switch (value) {
+    //     case "received" :
+    //         return history.pushs("/recieved");
+    //     case "sent" :
+    //         return history.pushs("/sent");
+    //     case "accepted" :
+    //         return history.pushs("/accepted");
+    //     default :
+    //         return history.pushs("/recieved");
```

Comment:
Can you please explain this code ?

I understood that you are trying to redirect to different pages.

But the function (as far as I can remember) is `history.push()`.... but not `history.pushs()`

Also, there is no need to return it in my opinion. Just run` history.push("/yourtargetpage")`

And... the most important point I want to mention is....

We only have ONE mailbox page where the user can view all interests (Received/Sent/Rejected.....)
So, why are you thinking of multiple pages ? This is not what we agreed upon... right?

Reply:
history.pushs() will be corrected. and it is not for the "Mailbox" page it for the "Matches" page. dont be confuse it with messages

Reply:
Ok. understood.